packages/server-admin-ui/src/views/appstore/Apps/Apps.js

```diff
@@ -176,11 +180,14 @@ const Apps = function (props) {
   )
 }
 
-const selectedViewToFilter = (selectedView) => {
+const selectedViewToFilter = (selectedView, appStore) => {
   if (selectedView === 'Installed') {
     return (app) => app.installing || app.installedVersion
   } else if (selectedView === 'Updates') {
-    return (app) => app.installedVersion && app.version !== app.installedVersion
+    return (app) =>
```

Comment:
Maybe extract this function as `updateAvailable(app, appStore)`? Use it in both places and include in the function a comment that that refers to PLUGINS_WITH_UPDATE_DISABLED? We managed to drop this functionality once, maybe this would guard against dropping in the future